middle-daemon/script/main.go

```diff
@@ -23,6 +24,13 @@ func main() {
 			check(err)
 
 			domain_name := words[0]
+			fmt.Println("Delivering precertificate...")
+			sourceFile := "../../CA-middle-daemon-storage/Precertificate/" + domain_name + "/precert.pem"
+			destinationfolder := "../../middle-daemon-website-daemon-storage/Precertificate/" + domain_name
+			if _, err := os.Stat(destinationfolder); os.IsNotExist(err) {
+				os.Mkdir(destinationfolder, 0700)
```

Comment:
```suggestion
				os.MkdirAll(destinationfolder, 0744)
```
Can we make all intermediate folders along the way with 744 mode so that other processes can see it?